service/worker/stream.go

```diff
@@ -38,23 +38,16 @@ func (ps *PushStream) Start() error {
 	var (
 		err error
 		// init these channels and pass to subscriber
-		// the lifecycle of these channels should be maintain by the user
+		// the lifecycle of these channels should be maintained by the user
 		subscriberRequestCh = make(chan *subscriber.PullRequest)
 		subscriberAckCh     = make(chan *subscriber.AckMessage)
 		subscriberModAckCh  = make(chan *subscriber.ModAckMessage)
 	)
 
-	ps.subs, err = ps.subscriberCore.NewSubscriber(ps.ctx, ps.nodeID, ps.subscriptionName, 100, 50, 0,
+	ps.subs, err = ps.subscriberCore.NewSubscriber(ps.ctx, ps.nodeID, ps.subscription.ExtractedSubscriptionName, 100, 50, 0,
```

Comment:
why extracted subscription name, full name will give project id as well. that will be easy to debug ?

Reply:
Sure. So use `ps.subscription.Key()` instead everywhere?

Reply:
Key will be our internal path 
/subscriptions/{projectid}/{extracted sub name}

we should log the public id i think

Reply:
logging `subscription.name` now